configs/pcx86/machine/compaq/lte286/cga/debugger/machine.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="/configs/pcx86/xsl/machine.xsl"?>
+<machine id="compaq-lte286-640k" type="pcx86" border="1" pos="center" background="default">
+	<name pos="center">COMPAQ LTE/286, 640Kb RAM</name>
+	<computer id="compaq-lte286-640k" name="COMPAQ LTE/286" buswidth="24"/>
+	<cpu id="cpu286" model="80286" cycles="12000000"/>
+	<ram id="ramLow" addr="0x00000" size="0xa0000"/>
+	<rom id="romBIOS" addr="0xf0000" size="0x10000" alias="0xff0000" file="/machines/pcx86/compaq/lte286/rom/1989-09-07/1989-09-07.json"/>
+	<video ref="/configs/pcx86/video/ibm/cga/ibm-cga.xml"/>
+	<fdc ref="/configs/pcx86/fdc/library.xml" autoMount='{A: {name: "SystemPro EISA Configuration 1.15"}}'/>
+	<!-- TODO: <hdc ref="/configs/pcx86/hdc/20mb/unformatted-at2.xml"/> -->
+	<keyboard ref="/configs/pcx86/keyboard/us84-softkeys.xml"/>
+        <debugger id="debugger" messages="" commands=""/>
+        <panel ref="/configs/pcx86/panel/wide286.xml"/>
+	<chipset id="chipset" model="deskpro386" floppies="[1440]" monitor="cga"/>
```

Comment:
I guess we need to add a new *model* at some point....

Reply:
> I guess we need to add a new _model_ at some point....

@jeffpar I plan to add more compaq machines after I finish with the LTE/286.  It may be nice to have a "vendor=(compaq|ibm|etc)" as I'm sure they will have a lot in common